src/utils/components/CloudinitModal/utils/cloudinit-utils.ts

```diff
@@ -114,7 +114,7 @@ export const updateCloudInitRHELSubscription = (
 
   const [cloudInitVol, restVolumes]: [null | V1Volume, V1Volume[]] = vmVolumes.reduce(
     (result, vol) => {
-      vol.cloudInitConfigDrive ? (result[0] = vol) : result[1].push(vol);
+      vol.cloudInitConfigDrive || vol.cloudInitNoCloud ? (result[0] = vol) : result[1].push(vol);
```

Comment:
Hmm, cannot we use `getCloudInitData` instead of `vol.cloudInitConfigDrive || vol.cloudInitNoCloud` here?

Reply:
we can thanks!